.eslintrc.cjs

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   extends: ['@theguild'],
+  parserOptions: {
+    project: './tsconfig.json',
```

Comment:
I removed rules that requires types information you need to update eslint config https://github.com/dotansimha/graphql-yoga/pull/2175

Reply:
I'll create another PR for that change because it caused some new errors when I updated.